examples/graphql_example/src/Plugin/GraphQL/DataProducer/QueryArticles.php

```diff
@@ -83,6 +83,7 @@ public function __construct(
    * @param \Drupal\Core\Cache\RefinableCacheableDependencyInterface $metadata
    *
    * @return \Drupal\graphql_examples\Wrappers\QueryConnection
+   *   Returns new QuerryConnection.
```

Comment:
Actually this is still not good. Return message must explain what is being returned in more detailed way than just saying what type it is returning. This is actually not a trivial cs fix, you need to understand the code. Try to read the code of every method and understand what is being returned. Then describe it in the return comment. For example here you need to understand what is QueryConnection obejct. You can go to it and see this is a custom wrapper upon Drupal's entity query. And here it is returned. After studying all of that we can finally make a good return comment:
```
   * @return \Drupal\graphql_examples\Wrappers\QueryConnection
   *   A query connection wrapper on top of Drupal's entity query to obtain
   *   the articles and their total amount.
```

Please continue in this way with other method comments. This module is quite robust and proper return comments are very important, so people can understand easily what is being done in particular method.